private/model/api/operation.go

```diff
@@ -271,6 +271,11 @@ func (e *example) traverseAny(s *Shape, required, payload bool) string {
 		str = e.traverseList(s, required, payload)
 	case "map":
 		str = e.traverseMap(s, required, payload)
+	case "jsonvalue":
+		str = "map[string]interface{}{}"
```

Comment:
I think this should be `aws.JSONValue{}` since moving to alias instead of map directly.